### PR TITLE
q:test2

### PR DESCRIPTION
**Pull request by q-rim**
Q made edits on your fille

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # hej-allihoppa
 Den första repositoryn nånsin!
-Nu har jag redigerat den här filen.
+Nu har jag redigerat den här filen.i
+
+Q:  test1
```
Comment (yongrim) [inline README.md]: can you change these to swedish? or german

**Comment by yongrim** [comment]
a few comments.

**Comment by q-rim** [comment]
wedf

**Comment by yongrim** [comment]
OK Im merging.

**Pull request merged**